docs/make.jl

```diff
@@ -4,6 +4,7 @@ push!(LOAD_PATH,"../src/")
 using Documenter
 using DocumenterCitations
 using QuantumSavory
+using QuantumSavory.ProtocolZoo
```

Comment:
It fixes this: https://github.com/QuantumSavory/QuantumSavory.jl/pull/107#issuecomment-2058456109

Reply:
Are you sure this was needed? Documentation building does not currently fail on master.

Reply:
This does cause documentation to fail here: https://github.com/QuantumSavory/QuantumSavory.jl/actions/runs/9554426617/job/26335434050?pr=120#step:7:708

Reply:
It would probably be better to mention `ProtocolZoo` here in the modules, I guess? https://github.com/Abhishek-1Bhatt/QuantumSavory.jl/blob/c591f6409295181026cfc473cdd7614f2fa819cf/docs/make.jl#L21

Reply:
yes, and in the using statement here https://github.com/Abhishek-1Bhatt/QuantumSavory.jl/blob/c591f6409295181026cfc473cdd7614f2fa819cf/docs/make.jl#L8

Reply:
This isn't helping the documentation build, looks like `using QuantumSavory.ProtocolZoo` is the only fix for it. It does not fail in master because master doesn't have the tutorial file: https://github.com/QuantumSavory/QuantumSavory.jl/pull/120/files#diff-3589a1552e319985a45bf5ee3739bdef54041717f875b1da1ecdfc90a4034d20
which was added here, and has the references to ProtocolZoo objects